pom.xml

```diff
@@ -1193,8 +1193,6 @@
                 </property>
             </activation>
             <modules>
-                <module>samples/client/petstore/typescript-axios/builds/with-npm-version</module>
-                <module>samples/client/petstore/typescript-axios/tests/default</module>
```

Comment:
@macjohnny FYI. I moved these tests from Travis CI to CircleCI as part of this PR.

Reply:
Great, thanks